tox.ini

```diff
@@ -10,6 +10,37 @@ requires =
 tox_pip_extensions_ext_venv_update = true
 tox_pyenv_fallback = false
 
+[pycodestyle]
+ignore =
+    # Disable pycodestyle errors and warnings that we don't care about because
+    # Black formats our code for us.
+    E203,  # Whitespace before ':',
+    E231,  # Missing whitespace after ',',
+    E501,  # Line too long,
+    W503,  # Line break before binary operator,
+
+    # Bare except. PyLint finds these for us so we don't need pycodestyle to.
+    E722,
```

Comment:
[pycodestyle supports reading config from `setup.cfg` or `tox.ini`](https://pycodestyle.pycqa.org/en/latest/intro.html#configuration) and we don't have a `setup.cfg`, so our pycodestyle is in `tox.ini`. This config still works if pycodestyle is run without tox: pycodestyle itself reads the `[pycodestyle]` section in the `tox.ini` file